functions/go/upsert-resource/metadata.yaml

```diff
@@ -5,6 +5,8 @@ tags:
 sourceURL: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/upsert-resource
 examplePackageURLs:
   - https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples/upsert-resource-simple
```

Comment:
I'd suggest to rename it to `upsert-resource-single`

Reply:
It is already out there. So better not rename it.